packages/helper-image/test/unit/spec/image-helper.browser.js

```diff
@@ -0,0 +1,137 @@
+/**!
+*
+* Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+*/
+
+import {assert} from '@ciscospark/test-helper-chai';
+import {readExifData, orient} from '../..';
+import file from '@ciscospark/test-helper-file';
+import sinon from '@ciscospark/test-helper-sinon';
+describe(`helper-image`, function() {
+  this.timeout(10000);
+
+  describe(`readExifData()`, () => {
+    it(`adds exif orientation information on the image file`, () => {
+      const sampleFile = {
+        displayName: `Portrait_7.jpg`,
+        fileSize: 405822,
+        type: `image/jpeg`,
+        image: {
+          height: 300,
+          width: 362
+        },
+        mimeType: `image/jpeg`,
+        objectType: `file`
+      };
+
+      return file.fetch(`/Portrait_7.jpg`)
+        .then((f) => {
+          readExifData(sampleFile, f)
+            .then((res) => {
+              assert.equal(res, f);
+              assert.equal(sampleFile.image.orientation, 7);
+            });
+        });
+    });
+  });
+
+  describe(`orient()`, () => {
+    const file = {
+      displayName: `Portrait_7.jpg`,
+      fileSize: 405822,
+      type: `image/jpeg`,
+      image: {
+        height: 300,
+        width: 362
+      },
+      mimeType: `image/jpeg`,
+      objectType: `file`
+    };
+    const options = {
+      img: `Portrait_7.jpg`,
+      x: 0,
+      y: 0,
+      width: 362,
+      height: 300,
+      ctx: {
+        save: sinon.stub().returns(() => true),
+        translate: sinon.stub().returns(() => true),
+        rotate: sinon.stub().returns(() => true),
+        scale: sinon.stub().returns(() => true),
+        drawImage: sinon.stub().returns(() => true),
+        restore: sinon.stub().returns(() => true)
+      }
+    };
+    [1, 2, 3, 4, 5, 6, 7, 8].forEach(/* eslint complexity: ["error", 9] */ (orientation) => {
```

Comment:
I don't think any of these tests run, at least not in a way that gets collected by the test reporter. take a look at https://github.com/ciscospark/spark-js-sdk/blob/master/packages/plugin-mercury/test/unit/spec/mercury-events.js#L188-L321 for an example of dynamic it statements